GitUI/Theming/ThemeLoader.cs

```diff
@@ -135,25 +143,47 @@ private static string[] GetClassNames(string themeFileName, StyleRule rule)
                 .Substring(ClassSelector.Length)
                 .Split(new[] { ClassSelector }, StringSplitOptions.RemoveEmptyEntries);
         }
+#endif
 
         private static Color GetColor(string themeFileName, StyleRule rule)
         {
-            if (rule.Declarations is null || rule.Declarations.Count != 1)
+            string cssColorValue = rule.Style.Color;
+            if (string.IsNullOrWhiteSpace(cssColorValue) || !cssColorValue.StartsWith("rgb("))
             {
                 throw StyleRuleThemeException(rule, themeFileName);
             }
 
-            var style = rule.Declarations[0];
-            if (style.Name != ColorProperty || !(style.Term is HtmlColor htmlColor))
+            // Wish we could write the following:
+            //
+            //      Property colorProperty = rule.Style.Declarations.FirstOrDefault(d => d.Name == ColorProperty);
+            //      Color color = ColorTranslator.FromHtml(colorProperty.DeclaredValue.Original.Text);
+            //
+            // colorProperty.DeclaredValue.Original.Text contains the required information, e.g. #f9f9f9
+            // but none of the properties nor types are public, thus leaving us with either atempting to
+            // access the data via the reflection, or parsing the raw text.
+            // Prefer the latter option - less magic.
```

Comment:
💡 It looks like better options are available in newer versions of ExCSS. For example:
https://github.com/TylerBrinks/ExCSS/blob/e6fec79cf7ba84c3a7ec2012aaeb2955a790ad98/src/ExCSS/Values/Color.cs

Reply:
I struggled to upgrade the code to the new version... Happy to look over it again once this is merged.

Reply:
I meant to say this would be a good follow-up issue. 👍